sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/AbstractTokenContextListener.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.appcenter.utils.context;
+
+/**
+ * Empty implementation to make callbacks optional.
+ */
+public class AbstractTokenContextListener implements AuthTokenContext.Listener {
```

Comment:
We use this pattern when there is more than 1 callback and that some of them are optional.
I don't see the use case for this one.